cfn/metrics/publisher_notag.go

```diff
@@ -143,7 +145,7 @@ func (p *Publisher) publishMetric(metricName string, data map[string]string, uni
 	}
 
 	pi := cloudwatch.PutMetricDataInput{
-		Namespace:  aws.String(p.namespace),
+		Namespace:  aws.String(fmt.Sprintf("%s/%s/%s", MetricNameSpaceRoot, p.providerAccountID, p.namespace)),
```

Comment:
if we set this in the constructor could just format once then save the namespace rather than formatting on every put

Reply:
Done